pkg/app/request/span.go

```diff
@@ -170,6 +171,13 @@ type GraphQL struct {
 	OperationType string `json:"operationType"`
 }
 
+type Elasticsearch struct {
+	DBCollectionName string `json:"dBCollectionName"`
+	// OperationBatchSize int    `json:"operationBatchSize"`
```

Comment:
Is this not used?

Reply:
I'm removing everything related to `OperationBatchSize`